addons/l10n_cl/views/report_invoice.xml

```diff
@@ -204,14 +223,30 @@
             <div t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'}"/>
         </xpath>
 
-        <xpath expr="//div[@id='total']" position="after">
+        <xpath expr="//div[@id='total']/div" position="inside">
+            <div name="stamp" class="col-11 text-center"/>
+        </xpath>
```

Comment:
This ensure correct dimensions of the stamp:
<img width="423" alt="Captura de pantalla 2023-10-02 a la(s) 09 33 58" src="https://github.com/odoo/odoo/assets/382893/cee5e979-d875-4a48-a6dc-bb5c23d22d2e">

<img width="1154" alt="Captura de pantalla 2023-10-02 a la(s) 09 40 17" src="https://github.com/odoo/odoo/assets/382893/61d7f855-5a47-4384-9299-2317d53ce21f">